internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceControllerWrapper.java

```diff
@@ -232,12 +231,13 @@ static StatefulServiceProvider<VeniceControllerWrapper> generateService(VeniceCo
           builder.put(KafkaTestUtils.getLocalCommonKafkaSSLConfig(SslUtils.getTlsConfiguration()));
         }
 
-        String fabricAllowList = "";
+        String fabricAllowList;
         if (options.isParent()) {
           // Parent controller needs config to route per-cluster requests such as job status
           // This dummy parent controller won't support such requests until we make this config configurable.
           // go/inclusivecode deferred(Reference will be removed when clients have migrated)
-          fabricAllowList = extraProps.getStringWithAlternative(CHILD_CLUSTER_ALLOWLIST, CHILD_CLUSTER_WHITELIST, "");
+          fabricAllowList =
+              extraProps.getStringWithAlternative(CHILD_CLUSTER_ALLOWLIST, CHILD_CLUSTER_WHITELIST, StringUtils.EMPTY);
```

Comment:
Hmm... is it useful to use this constant? What's the difference?

Reply:
Yes, they are the same. Just treat it as a predefined constant value with a bit more expressive name to me and refer to it. I don't mind reverting it back if `""` looks obvious enough.

Reply:
It's fine either way, as you wish.